src/main/java/com/moodybluez/enterprise/dto/Date.java

```diff
@@ -5,6 +5,7 @@
 public @Data
 class Date {
     private String date;
-    private int dayOfWeekID;
-    private String dayOfWeekDesc;
+    private int dateID;
+    private int weekDayID;
+    private String weekDay;
```

Comment:
Updated to more generic naming Convention format. Especially the description variable name because it is confusing. You are already using a "meaning" of word "description" in your metrics page, so here it could confuse someone like me as you are referring this to basically the day "name" like "Monday".
Citation - "a name that tells the reader of the program what the variable represents, Keep the variable names short avoid including metadata"  - [R. Hamedy](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)